apps/react-18-tests-v8/webpack.config.js

```diff
@@ -12,22 +12,53 @@ module.exports = () => {
     },
     resolve: {
       extensions: ['.tsx', '.ts', '.js'],
-      plugins: [
-        new TsconfigPathsPlugin({
-          configFile: path.resolve(__dirname, '../../tsconfig.base.json'),
-        }),
-      ],
       alias: {
         react: path.resolve(__dirname, './node_modules/react'),
         'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
+        ...getResolveAlias(),
       },
     },
     module: {
       rules: [
         {
           test: /\.tsx?$/,
           exclude: /node_modules/,
-          use: 'swc-loader',
+          use: {
```

Comment:
v8 cannot be compiled with swc, switching to esbuild for now https://github.com/swc-project/swc/issues/2037